src/app/shared/forms/planet-tag-input.component.ts

```diff
@@ -215,14 +215,11 @@ export class PlanetTagInputComponent implements ControlValueAccessor, OnInit, On
     }
   }
 
-  addTagCount(tags, selectMany = false) {
+  addTagCount(tags) {
     return this.mode === 'add' ? tags : tags.map((tag) => {
-      return !selectMany ? tag : ({
+      return ({
         ...tag,
-        count: this.filteredData.reduce(
-          (count, item: any) => count + ((item.tags || []).findIndex((itemTag: any) => itemTag._id === tag._id) > -1 ? 1 : 0),
-          0
-        )
+        count: this.filteredData.length
```

Comment:
This just shows the same number for each collection.

You should filter the `filteredData` for items that have the `tag` and then take the length of that.